### PR TITLE
feat: Add devtools panel resize

### PR DESCRIPTION
**Pull request by ptbrowne**
Hurray, we can resize the devtools panel.
Logic taken from react-query :blush:.

### CODE REVIEW

File: packages/cozy-client/types/types.d.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ export type ClientResponse = any;
 export type Manifest = any;
 export type OldCozyClient = any;
 export type NodeEnvironment = any;
-export type QueryFetchStatus = "failed" | "loading" | "pending" | "loaded";
+export type QueryFetchStatus = "loading" | "loaded" | "pending" | "failed";
```
Comment (Crash--) [inline packages/cozy-client/types/types.d.ts:22]: What do you do with this file?! You don't like it? You edit it every time! 🤣

File: packages/cozy-client/src/devtools/index.jsx [hunk 2]
```diff
@@ -78,35 +81,101 @@ const DevToolsNavList = ({ selected, panels, onNav }) => {
...
+    }
+
+    const unsub = () => {
+      document.removeEventListener('mousemove', run)
+      document.removeEventListener('mouseUp', unsub)
+    }
+
+    document.addEventListener('mousemove', run)
+    document.addEventListener('mouseup', unsub)
+  }
```
Comment (Crash--) [inline packages/cozy-client/src/devtools/index.jsx:145]: I had something similar in mind, but this code is much more smaller than I excepted! Well done

**Review by Crash-- — APPROVED** [review]

File: packages/cozy-client/src/devtools/index.jsx (hunk 2 above)
Comment (ptbrowne) [reply packages/cozy-client/src/devtools/index.jsx:145]: All credits goes to the initial implementation inside react-query :)

File: packages/cozy-client/types/types.d.ts (hunk 1 above)
Comment (ptbrowne) [reply packages/cozy-client/types/types.d.ts:22]: It's typescript that does that, it's pretty annoying.

**Pull request merged**